tasks/development.ts

```diff
@@ -205,4 +221,165 @@ export function load() {
     await run(ethsAddress, wrbtcAddress, "eth", "btc");
     await run(rusdtAddress, wrbtcAddress, "usd", "btc");
   });
+
+  task("deploy-timelock")
```

Comment:
```suggestion
  task("dev:deploy-timelock")
```

Also change it on the readme file